packages/logs/src/boot/startLogs.ts

```diff
@@ -88,6 +91,29 @@ export function doStartLogs(
   }
   errorObservable.subscribe(reportError)
 
+  if (isExperimentalFeatureEnabled('forward-reports')) {
+    trackReports(['deprecation', 'intervention'], (message: string, report: Report) => {
+      const loggerMethod = report.type === 'deprecation' ? 'warn' : 'error'
+      errorLogger[loggerMethod](message, { report: JSON.parse(JSON.stringify(report)) })
+    })
+    trackCspViolation((message: string, event: SecurityPolicyViolationEvent) => {
+      errorLogger.error(message, {
+        report: {
+          blockedURI: event.blockedURI,
+          columnNumber: event.columnNumber,
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** what is the reasoning for mapping each attributes individually? could we do `report: event`?‏

Reply:
I tried. With our `jsonStringify` function or the native `JSON.stringify` I only get `{"isTrusted":true}`